docker-compose.yaml

```diff
@@ -26,10 +26,16 @@ services:
         - A_PROVIDER_PORT=3002
         - A_VITE_HOST=localhost
         - A_VITE_PORT=3000
+        - A_DB_HOST=utility_db
+        - A_DB_PORT=5432
+        - A_DB_PASSWORD=utility_password
```

Comment:
For them to actually work, please also add envs/args to BackEnd/Dockerfile, should be trivial enough (just repeat after wha already exists)

Reply:
Fixed